### PR TITLE
Adding more details to one-link answers of the FAQ

### PR DESCRIPTION
**Pull request by BalmungSan**

### CODE REVIEW

**Review by SethTisue — APPROVED** [review]

File: _overviews/FAQ/index.md [hunk 1]
```diff
@@ -81,14 +81,13 @@ See the [Scala 2.13 Collections Guide](https://docs.scala-lang.org/overviews/col
 
 ### What are context bounds (`[T : Foo]`)?
 
-Basically is sugar syntax for an `implicit` argument of type `Foo[T]`.
+It's syntactic sugar for an `implicit` parameter of type `Foo[T]`.
 
-More deatils in this [Stack Overflow answer](https://stackoverflow.com/a/4467012).
+More details in this [Stack Overflow answer](https://stackoverflow.com/a/4467012).
```
Comment (BalmungSan) [inline _overviews/FAQ/index.md:86]: Sorry about this :flushed:

File: _overviews/FAQ/index.md [hunk 2]
```diff
@@ -155,7 +154,7 @@ and the [typeclass pattern](http://tpolecat.github.io/2013/10/12/typeclass.html)
 
 This [blog post](http://tpolecat.github.io/2015/04/29/f-bounds.html)
 argues against F-bounds and in favor of typeclasses.
-Which is revisited with a discussion about the trade-offs of alternatives
+Another discussions of trade-offs and alternatives is
```
Comment (BalmungSan) [inline _overviews/FAQ/index.md:157]: I mean to say that the SO post discuss trade-offs between all alternatives.
While I like your wording more than mine, I am not totally sure if it gives the correct idea.

Comment (SethTisue) [reply _overviews/FAQ/index.md]: I pushed another commit with an adjustment; wdyt?

Comment (SethTisue) [reply _overviews/FAQ/index.md]: I like how you have marked your link answers as "community wiki", btw 👍

Comment (BalmungSan) [reply _overviews/FAQ/index.md]: @SethTisue It looks good to me!

**Pull request merged**